docs/en/application.md

```diff
@@ -154,4 +154,50 @@ After the `Application` resource is applied to Kubernetes cluster, the KubeVela
 
 > TBD: the revision names and labels for resource instances are currently work in progress.
 
-> TBD: a demo for kubectl apply above Application CR and show full detailed underlying resources.
+## Run Application
+You should make sure those workloads and traits definition installed already, you can find sidecar trait [here](../examples/registry/sidecar.yaml).
```

Comment:
please change `../examples/registry/sidecar.yaml`  to  `https://github.com/oam-dev/kubevela/tree/master/docs/examples/registry/sidecar.yaml`